### DISCUSSION TITLE
Is BIIGLE down?

### DISCUSSION THREAD

**Discussion by PoppyKeogh** [biigle/core#481] [Q&A] [answered]
Hello,

I'm experiencing issues with loading the BIIGLE website. I have been using BIIGLE on and off for the last year and have not had this issue before. I check my firewall setting and ran diagnostics and the issue seems to be with the website itself. The error is pasted below. Any help on this would be greatly appreciated.

This site can’t be reachedbiigle.de took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_TIMED_OUT_

**Comment by mzur** [comment, marked as answer]
Hi Poppy, our compute cloud was shut down for maintenance yesterday. You can follow the updates regarding BIIGLE here: https://status.biigle.de/

BIIGLE will be available again on Thursday.

**Reply by PoppyKeogh** [reply]
Hi Martin,

Thanks for this info, I had checked the status updates but must have missed the latest one.

**Reply by mzur** [reply]
We are working on a system to better communicate important news like this. So in the future hopefully less users are caught by surprise.

**Reply by goldsoundz80** [reply]
Ahh glad I found this, I've been trying to reach the site for a couple of days to access project work (trying periodically every hour or so!). It would be useful to have the www.biigle.de homepage redirect to the status page when under maintenance.
Thanks, Rob
